apps_group_PR_model.md

```diff
@@ -0,0 +1,51 @@
+## Problem
+- *Description of why you made this PR*
+
+## Solution
+- *And how you fix that*
+
+## PR Status
+*Obviously, you should really check these affirmations*  
+Work finished. Package_check, basic tests and upgrade from last version OK.  
+Could be reviewed and tested.
+
+## Validation
+---
+*Minor decision*
+- [ ] **Upgrade previous version** : 
+- [ ] **Code review** : 
+- [ ] **Approval (LGTM)** : 
+- [ ] **Approval (LGTM)** : 
+- [ ] **CI succeeded** : 
+When the PR is mark as ready to merge, you have to wait for 3 days before really merge it.
+
+*Medium decision*
+- [ ] **Complete test** : 
+- [ ] **Upgrade previous version** : 
+- [ ] **Code review** : 
+- [ ] **Code review** : 
+- [ ] **Approval (LGTM)** : 
+- [ ] **Approval (LGTM)** : 
+- [ ] **Approval (LGTM)** : 
+- [ ] **CI succeeded** : 
+- [ ] **CI succeeded** : 
```

Comment:
Ok I have my answer, it's the next sentences : "One public package check log is enough instead of two checks. "